src/runtime_src/core/edge/common/aie_parser.cpp

```diff
@@ -446,6 +483,19 @@ get_tiles(const xrt_core::device* device, const std::string& graph_name)
   return ::get_tiles(aie_meta, graph_name);
 }
 
+std::vector<tile_type>
+get_event_tiles(const xrt_core::device* device, const std::string& graph_name,
+                e_tile_type type)
+{
+  auto data = device->get_axlf_section(AIE_METADATA);
+  if (!data.first || !data.second)
+    return std::vector<tile_type>();
```

Comment:
`return {};`

Reply:
Curious why this was not accepted?

Reply:
This file is maintained by another development team, and my return statement is consistent with all other functions.